templates/galaxy/config/container_resolvers_conf.yml.j2

```
@@ -0,0 +1,128 @@
+# Sample file for container resolvers. Resolvers that are potentially used by default
+# are uncommented (comments describe under which premises they are in the
+# defaults).
+
+# Note that commented yaml does not have a space after the #
+# while additional explanations do.
+
+# Explicit container resolvers
+# ============================
+
+# get a container description (URI) for an explicit docker container requirement
+- type: explicit
+
+# get a container description (URI) for an explicit singularity container requirement
+- type: explicit_singularity
+
+# get a cached container description (path) for singularity
+# pulls the container into a cache directory if not yet there
+#- type: cached_explicit_singularity
+  # set the cache directory for storing images
+  #cache_directory: database/container_cache/singularity/explicit
+
+# Mulled container resolvers
+# ==========================
+
+# The following uncommented container resolvers are in the defaults
+# if ``enable_mulled_containers`` is set in ``galaxy.yml`` (which is the default).
+
+# get a container description to a cached mulled docker container
+- type: cached_mulled
+  # This resolver type will check the image cache for `quay.io/NAMESPACE/containername`
+  # The image cache can be populated by `mulled_*` resolvers or
+  # with a manual `docker pull quay.io/NAMESPACE/containername`.
+  namespace: biocontainers
+#- type: cached_mulled
+#  namespace: local
+#  # The local namespace refers to locally built images that are prefixed with quay.io/local/.
+#  #namespace: biocontainers
+#  # the version of the mulled hashing function (v2/v1)
+#  #hash_func: v2
+
+
+
+# checks if the image file exists in `cache_directory`
+- type: cached_mulled_singularity
+  #namespace: biocontainers
+  cache_directory: "/cvmfs/singularity.galaxyproject.org/all/"
+  #
+  # the method for caching directory listings (not the method for image caching)
+  # can be uncached or dir_mtime (the latter only determines the directory listing
+  # if the modification time of the directory changed)
+  #cache_directory_cacher_type: uncached
+
+#- type: cached_mulled_singularity
+#  namespace: local
+#  #cache_directory_cacher_type: uncached
+
+# Resolves container images from quay.io/NAMESPACE/MULLED_HASH where the
+# mulled hash describes which packages and versions should be in the container
+#
+# If the docker CLI is available the image will be pulled.
+#
+# These resolvers are generally listed after the cached_* resolvers, so that images
+# are not pulled if they are already cached.
+- type: mulled
+  namespace: biocontainers
+  #
+  # If true return quay.io/NAMESPACE/MULLED_HASH otherwise to the cached image
+  # which is also quay.io/NAMESPACE/MULLED_HASH in case of docker
+  #auto_install: true
```

```suggestion
  auto_install: true
```

We have auto install explicitly set to `true` in the XML version of the file, although judging by the description, I don't even think it makes a difference.